uk.co.compucorp.civicrm.hrcore/templates/CRM/HRCore/UserMenuMarkup.tpl

```diff
@@ -10,6 +9,9 @@
   </div>
   <nav class="chr_user-menu__dropdown">
     <ul>
+      <li class="chr_user-menu__dropdown__username">
+        <span>signed in as <strong>{$username}</strong></span>
```

Comment:
This instantly caught my attention when I looked at GIFs, I think "signed" should be capitalised.